system-probe_x64/Dockerfile

```diff
@@ -25,6 +28,8 @@ RUN apt-get update && apt-get install -y --no-install-recommends apt-utils && ap
         libxml2-dev \
         libzip-dev \
         linux-headers-amd64 \
+        linux-headers-5.10.0-0.bpo.12-amd64 \
```

Comment:
Since the docker image isn't pinned a specific version of `buster`, won't this specific version become outdated?

Reply:
It may indeed be the case. I could not find a reliable way to get the latest kernel headers. Maybe using somethink like linux-headers-5*-amd64 would be better ?